repobee_sanitizer/_sanitize.py

```diff
@@ -34,6 +34,23 @@ def sanitize_file(file_abs_path: pathlib.Path) -> Optional[str]:
         return "\n".join(sanitized_string)
 
 
+def strip_file(file_abs_path: pathlib.Path) -> Optional[str]:
+    """Runs the strip protocol on a given file. This will remove all sanitizer
+    markers from the file. File must be syntax checked before running this.
+
+    Args:
+        file_abs_path: The absolute file path to the file you wish to
+        sanitize.
```

Comment:
Incorrectly formatted, the second line will be interpreted as a new parameter.
```suggestion
        file_abs_path: The absolute file path to the file you wish to
            sanitize.
```